tests/integration_tests/functional/test_new_tap.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Check that network is working after restore with a different tap."""
+import pytest
+from integration_tests.functional.test_cpu_features import PLATFORM
+from framework.artifacts import SnapshotType
+from framework.builder import SnapshotBuilder, MicrovmBuilder
+import framework.utils as utils
+import host_tools.network as net_tools
+
+
+@pytest.mark.skipif(
+    PLATFORM != "x86_64",
+    reason="There is no real need to test this on both platforms."
+)
+def test_snapshot_delete_tap(bin_cloner_path):
+    """Test that network works after restore with a different tap.
```

Comment:
Isn't this covered in other tests, like https://github.com/firecracker-microvm/firecracker/blob/9e3c10fcd7429f748322080e41f7562917cc1978/tests/integration_tests/functional/test_snapshot_advanced.py#L216 ?

Reply:
removed the integration test